src/main/java/org/hydev/mcpm/client/adaptor/DownloaderProgressBar.java

```diff
@@ -0,0 +1,26 @@
+package org.hydev.mcpm.client.adaptor;
+
+import org.hydev.mcpm.client.interaction.ProgressBar;
+import org.hydev.mcpm.client.interaction.ProgressBarTheme;
+import org.hydev.mcpm.client.interaction.ProgressRowBoundary;
+
+import java.util.List;
+
+public class DownloaderProgressBar extends ProgressBar {
+    /**
+     * Create and initialize a progress bar
+     *
+     * @param theme Selected theme
+     */
+    public DownloaderProgressBar(ProgressBarTheme theme) {
+        super(theme);
+
+    }
+
+    @Override
+    public void incrementBarProgress(int id, long inc) {
+        List<ProgressRowBoundary> bars = getBars();
+        bars.get(id).increase(inc);
+    }
+
+}
```

Comment:
why are separate progress bars needed?

Reply:
Oh. I actually didn't use it. I was planning to change some of the speed, but I don't need it anymore. I will delete this.

Reply:
okay great!